Formula/node.rb

```diff
@@ -1,8 +1,8 @@
 class Node < Formula
   desc "Platform built on V8 to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v13.11.0/node-v13.11.0.tar.xz"
-  sha256 "e5402183e68806785b3c40c2cb0a6b6aa43bb61aee1cec5efde9c65825ef291f"
+  url "https://nodejs.org/dist/v13.12.0/node-v13.12.0.tar.gz"
```

Comment:
```suggestion
  url "https://nodejs.org/dist/v13.12.0/node-v13.12.0.tar.xz"
```

Can we stick with `xz`?

Reply:
yeah, my bad